### ISSUE TITLE
?? errors on builtins

### ISSUE THREAD

**Issue by davidszotten** [pdbpp/pdbpp#286]
```
(Pdb++) [].clear??
Type:           builtin_function_or_method
String Form:    <built-in method clear of list object at 0x102702308>
Definition:     [].clear()
Docstring:      Remove all items from list.
Source:
** Error: module, class, method, function, traceback, frame, or code object was expected, got builtin_function_or_method **
```

**Comment by blueyed** [comment]
Yeah, this could be just a note maybe, on the same line as "Source:" probably.

**Comment by blueyed** [comment]
Fixed, likely/maybe f72e48c.
